update note about root certificate (#5922)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-pki.md

The certificate template is configured to supersede all the certificate templates provided in the certificate templates superseded templates list.  However, the certificate template and the superseding of certificate templates is not active until you publish the certificate template to one or more certificate authorities.

> [!NOTE]

> > The root certificate of the certification authority must be present in the NTAuth store. By default, the Active Directory Certificate Authority's root certificate is added to the NTAuth store. If you are using a 3rd party CA, this may not be done by default. If the certificate authority's root certificate is not present in the NTAuth store, user authentication will fail.

Reviewer: ```suggestion
> The domain controller's certificate must chain to a root in the NTAuth store. By default, the Active Directory Certificate Authority's root certificate is added to the NTAuth store. If you are using a 3rd party CA, this may not be done by default. If the domain controller certificate does not chain to a root in the NTAuth store, user authentication will fail.
```

Reviewer 2: Please change "3rd party" to "third-party" in this suggestion.

Reviewer 3: Both suggestions merged:
```suggestion
> The domain controller's certificate must chain to a root in the NTAuth store. By default, the Active Directory Certificate Authority's root certificate is added to the NTAuth store. If you are using a third party CA, this may not be done by default. If the domain controller certificate does not chain to a root in the NTAuth store, user authentication will fail.

```

The certificate template is configured to supersede all the certificate templates provided in the certificate templates superseded templates list.  However, the certificate template and the superseding of certificate templates is not active until you publish the certificate template to one or more certificate authorities.

> >[!NOTE]

Reviewer 3: Oh, right. I almost forgot this one. MD compatibility spacing (as below).
```suggestion
> [!NOTE]
```

> >The root certificate of the certification authority must be present in the NTAuth store. By default, Active Directory Certificate Authority's root certificate is added to the NTAuth store. If you are using a 3rd party CA, this may not be done by default. If the certificate authority's root certificate is not present in the NTAuth store, user authentication will fail. 

Reviewer 3: Looks quite good, although sentence 2 needs the same definite article ("the") as used in sentence 3:
```suggestion
> The root certificate of the certification authority must be present in the NTAuth store. By default, the Active Directory Certificate Authority's root certificate is added to the NTAuth store. If you are using a 3rd party CA, this may not be done by default. If the certificate authority's root certificate is not present in the NTAuth store, user authentication will fail.
```
(also added MD quote indent space and removed 1 whitespace at the end)
